wiki/Help_Centre/Account/id.md

```diff
@@ -0,0 +1,235 @@
+
+Berikut merupakan beberapa contoh informasi yang dapat membantu kami dalam memulihkan akunmu:
+
+- Alamat email yang kamu gunakan pertama kali untuk mendaftarkan akunmu (serta alamat-alamat email lainnya yang berhubungan)
+- Bukti-bukti pembelian osu!supporter tag (atau barang-barang lainnya dari osu!store) yang kamu miliki
+- Tanggal terakhir kamu mengakses akunmu
+- Informasi-informasi lainnya yang kamu rasa dapat membantu (kecuali skor dan *replay*)
+
+### Akun saya telah dicuri oleh pihak lain!
+
+**Harap periksa apakah kamu mampu untuk mengatur ulang kata sandi milikmu melalui [laman pemulihan kata sandi berikut](https://osu.ppy.sh/home/password-reset).**
+
+Apabila kamu mampu, ikuti instruksi-instruksi yang tertera pada laman yang bersangkutan untuk dapat memulihkan akunmu sepenuhnya.
+
+Sebaliknya, apabila kamu tidak mampu, harap segera hubungi layanan dukungan kami melalui email pada alamat [accounts@ppy.sh](mailto:accounts@ppy.sh) dengan mencantumkan informasi selengkap-lengkapnya seputar akun yang kamu miliki.
+
+Kami sangat menyarankan kamu untuk menghubungi kami melalui alamat email yang kamu pertama kali gunakan untuk mendaftarkan akunmu. Di samping itu, kami juga akan sangat terbantu apabila kamu dapat menyertakan informasi seputar hal-hal yang sekiranya hanya dapat diketahui olehmu dan tim kami.
+
+Harap untuk tidak menyertakan informasi seputar skor-skor dan *replay-replay* yang pernah kamu torehkan - informasi-informasi demikian pada umumnya sangat rawan dipalsukan dan tidak dapat kami gunakan sebagai bukti yang kuat untuk memulihkan akunmu.
+
+### Mengapa saya terus-menerus mendapatkan pesan 'Bancho authentication error'?
+
+**Pesan tersebut pada umumnya muncul akibat terdapatnya suatu masalah jaringan yang membuat klien osu! milikmu tidak dapat terhubung ke server yang kami miliki.**
+
+Terdapat beberapa hal yang dapat kamu coba untuk menyelesaikan masalah ini:
+
+#### Apakah kamu sudah memasukkan kata sandi milikmu dengan benar?
+
+Apabila kamu melupakan kata sandimu, kamu dapat mengatur kata sandi baru pada [laman pemulihan kata sandi berikut](https://osu.ppy.sh/home/password-reset).
+
+#### Apakah kamu sedang berada di dalam suatu *proxy server* atau jaringan VPN?
+
+**Cobalah untuk keluar dari *proxy server* atau jaringan VPN yang sedang kamu gunakan.**
+
+Sepengetahuan kami, terdapat beberapa penyedia jasa layanan *proxy server* dan jaringan VPN yang koneksinya tidak kompatibel dengan osu! karena satu dan lain hal.
+
+### Saya telah mencoba solusi-solusi di atas, namun saya masih mendapatkan pesan error yang sama!
+
+**Harap kirimi kami berkas `network.log` milikmu agar kami dapat membantumu lebih lanjut.**
+
+Untuk mengetahui cara mendapatkan berkas `network.log` yang dimaksud, harap lihat [Berkas-berkas log](/wiki/id/Help_Centre#Berkas-berkas_log).
```

Comment:
```suggestion
Untuk mengetahui cara mendapatkan berkas `network.log` yang dimaksud, harap lihat [Berkas-berkas log](/wiki/Help_Centre).
```

Reply:
ah, this url will actually link to the correct place once #5696 is merged :

![image](https://user-images.githubusercontent.com/36564236/123534198-88a28680-d745-11eb-871e-4548489b6e93.png)

![image](https://user-images.githubusercontent.com/36564236/123534213-b1c31700-d745-11eb-9f41-80b7288e38f8.png)